spec/factories/info_requests.rb

```diff
@@ -6,34 +6,34 @@
     public_body
     user
 
-    after_create do |info_request, evaluator|
-      FactoryGirl.create(:initial_request, :info_request => info_request)
+    after(:create) do |info_request, evaluator|
+      create(:initial_request, :info_request_id => info_request.id)
```

Comment:
It was causing an odd deprecation warning along the lines of

```
DEPRECATION WARNING: Accessing attributes from initialize_with when duplicate assignment is enabled is deprecated; use FactoryGirl.duplicate_attribute_assignment_from_initialize_with = false.
```

...but that might have been specific to one of the 3.x.x branches I needed to go through to get to 4.7.0. I can put it back if you'd prefer. In fact as I've just put it back to test whether it breaks, might as well add a fixup commit for it